vpr/src/base/vpr_types.h

```diff
@@ -1271,7 +1271,8 @@ struct t_analysis_opts {
 
 // used to store NoC specific options, when supplied as an input by the user
 struct t_noc_opts {
-    bool noc; ///<options to model the noc within the FPGA device
+    bool noc;                   ///<options to model the noc within the FPGA device
```

Comment:
Turns on hard NoC modeling & optimization 
(update text in comment).

Reply:
Done